jetty-server/src/main/java/org/eclipse/jetty/server/handler/DefaultHandler.java

```diff
@@ -116,62 +119,87 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
         response.setStatus(HttpServletResponse.SC_NOT_FOUND);
         response.setContentType(MimeTypes.Type.TEXT_HTML.toString());
```

Comment:
Given that you're using UTF-8, here you should use the `TEXT_HTML_UTF_8` constant instead.

Reply:
done